base/docs/helpdb.jl

```diff
@@ -6038,34 +6013,32 @@ with appropriate ``op`` values:
 manage
 
 doc"""
-```rst
-..  resize!(collection, n) -> collection
+    resize!(collection, n) -> collection
 
-Resize ``collection`` to contain ``n`` elements.
-If ``n`` is smaller than the current collection length, the first ``n``
-elements will be retained. If ``n`` is larger, the new elements are not
+Resize `collection` to contain `n` elements.
+If `n` is smaller than the current collection length, the first `n`
+elements will be retained. If `n` is larger, the new elements are not
 guaranteed to be initialized.
 
-.. doctest::
-
-  julia> resize!([6, 5, 4, 3, 2, 1], 3)
-  3-element Array{Int64,1}:
-   6
-   5
-   4
-
-.. code-block:: julia
+```jldoctest
+julia> resize!([6, 5, 4, 3, 2, 1], 3)
+3-element Array{Int64,1}:
+ 6
+ 5
+ 4
+```
 
-  julia> resize!([6, 5, 4, 3, 2, 1], 8)
-  8-element Array{Int64,1}:
-   6
-   5
-   4
-   3
-   2
-   1
-   0
-   0
+```jl
```

Comment:
It wasn't originally.  I believe the final two zeros are uninitialized, so it's not a deterministic test.